src/verify_cert.rs

```diff
@@ -139,6 +139,7 @@ impl<'a, 'p: 'a> ChainOptions<'a, 'p> {
         path: &PathNode<'_>,
         trust_anchor: &TrustAnchor,
         budget: &mut Budget,
+        time: UnixTime,
```

Comment:
Nit: would prefer to order `time` between `path` and `trust_anchor` instead of last, fitting the most-to-least-specific order of arguments.